certified-connectors/Kapish Power Connect/apiDefinition.swagger.json

```diff
@@ -0,0 +1,8957 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Micro Focus Content Manager",
+    "description": "Micro Focus Content Manager is a governance-based enterprise content management system designed to help government agencies, regulated industries and global organizations manage their business content from creation to disposal. Connect to Micro Focus Content Manager to perform actions including creating or updating records or locations, uploading or downloading electronic files, searching and more. (1.0.2.712)",
+    "version": "1.0.2.712",
```

Comment:
@BrettHankinson since this is a new connector, can you please change the version to "1.0"?